src/components/organisms/ChatSidebar/components/RecipientChat/RecipientChat.scss

```diff
@@ -19,6 +19,12 @@
     cursor: pointer;
   }
 
+  &__chatbox {
+    position: absolute;
+    top: 45px;
+    height: calc(100% - 45px);
```

Comment:
move the 45px into a const and reuse it here

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/2ed1728e022e999af48c594fbbe07474088367be